tests/src/Context/DatabaseLogContext.php

```diff
@@ -65,4 +84,31 @@ protected function assertMessageLogged($severity, $type, $message) {
     assert($full_message, matches('@' . $message . '@'));
   }
 
+  /**
+   * Assert that a specific message is not logged.
+   *
+   * @param int $severity
+   *   The severity of the message.
+   * @param string $type
+   *   The type (category) of the message.
+   * @param string $message
+   *   A regular expression pattern the message needs to match.
+   *
+   * @see watchdog()
+   */
+  protected function assertMessageNotLogged($severity, $type, $message) {
+    $query = db_select('watchdog', 'w');
```

Comment:
We could create a function with this part of code because it's used on two functions.(assertMessageLogged and assertMessageNotLogged)